Source/WebCore/Modules/webaudio/AudioWorkletGlobalScope.cpp

```diff
@@ -144,6 +144,10 @@ RefPtr<AudioWorkletProcessor> AudioWorkletGlobalScope::createProcessor(const Str
         return nullptr;
 
     JSC::JSObject* jsConstructor = constructor->callbackData()->callback();
+    ASSERT(jsConstructor);
+    if (!jsConstructor)
+        return nullptr;
+
```

Comment:
Is this change meant to be part of this patch?

Reply:
Nope, this was part of a different commit that got swept up in my initial PR.